ang/civicase/Dashboard.html

```diff
@@ -0,0 +1,149 @@
+                </div>
+                <div class="panel-body"><div civicase-type-donut-chart="{my_cases: myCasesOnly, 'status_id.grouping': 'Closed'}"></div></div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ts('Average Duration')}}</h3>
+                </div>
+                <div class="panel-body"><div civicase-type-duration-chart="{my_cases: myCasesOnly}"></div>
+              </div>
+            </div>
+          </div>
+          </div>
+
+          <div class="row dashboard-case-wrapper">
+            <div class="col-md-4">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ ts('Next Milestones') }}
+                    <span class="badge badge-primary" ng-class="{'badge-danger': dashboardActivities.nextMilestones[0].is_overdue}">{{ dashboardActivities.nextMilestonesCount }}</span>
+                    <a ng-href="{{ seeAllLink('milestone', 'incomplete') }}" class="pull-right">({{ ts('see all') }})</a>
+                  </h3>
+
+                </div>
+                <div class="panel-body">
+                  <div>
+                    <ul class="list-group act-feed-list-group">
+                      <li class="list-group-item" ng-repeat="activity in dashboardActivities.nextMilestones">
+                        <div
+                          case-activity-card="activity"
+                          refresh-callback="refresh"
+                          edit-activity-url="editActivityUrl"
+                        ></div>
+                      </li>
+                      <li class="list-group-item" ng-repeat="placeholder in activityPlaceholders" ng-if="!dashboardActivities.nextMilestones">
+                        <div class="civicase__loading-placeholder__activity-card"><div></div></div>
+                      </li>
+                    </ul>
+                    <div class="civicase__panel-empty" ng-if="dashboardActivities.nextMilestones && !dashboardActivities.nextMilestones.length">
+                      <i class="fa fa-flag fa-big"></i>
+                      <div class="empty-label">{{ ts('There are currently no milestones') }}</div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h3 class="panel-title">{{ ts('Recent Communications') }}
+                    <span class="badge badge-primary">{{ dashboardActivities.recentCommunicationCount }}</span>
+                    <a ng-href="{{ seeAllLink('communication', 'completed') }}" class="pull-right">({{ ts('see all') }})</a>
+                  </h3>
+                </div>
+                <div class="panel-body">
+                  <div>
```

Comment:
Any specific reason for extra div ?

Reply:
The whole file has been renamed, thats why everything is showing as changed.
But this HTML is part of Dashboard tab, which is not touched in this PR.

It needs to be taken care of when we work on the Dashboard tab ticket.